app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -44,6 +45,15 @@ class LuciBuildService {
     this.log = log;
   }
 
+  /// Shards [rows] into several sublists of size [maxEntityGroups].
```

Comment:
Shall we define the `maxEntityGroups` as a const variable?

Reply:
Not in here, but the calling function is already using a configuration value.